app/reducers/groups.js

```diff
@@ -31,6 +31,11 @@ export default createEntityReducer({
         newState.byId[action.meta.groupId].memberships.push(
           action.payload.result
         );
+        if (
+          typeof newState.byId[action.meta.groupId].numberOfUsers === 'number'
```

Comment:
This will btw. never be 100% correct, since the count is for all groups below as well, but this is a ish. heuristic.

Reply:
Why don't we just refetch the group when adding a membership?

Reply:
We would then need to refetch all groups, since this is the members in the group and all its children. I think this is ok for now.

Reply:
hmm, I see. I guess it's fine then